pypowermate/__init__.py

```diff
@@ -1 +1 @@
-from powermate import Powermate, PowermateTimeoutException
```

Comment:
Why?

Reply:
Sorry for the delay.

So If there is `from powermate import Powermate, PowermateTimeoutException`, error is :
```Python 3.4.2 (default, Oct  8 2014, 10:45:20) 
>>> import pypowermate
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ratnayake/Documents/projets/audionet/powermate/pypowermate-master/pypowermate/__init__.py", line 1, in <module>
    from powermate import Powermate, PowermateTimeoutException
ImportError: No module named 'powermate'
```


If there is `from pypowermate import Powermate, PowermateTimeoutException`, error is :
```Python 3.4.2 (default, Oct  8 2014, 10:45:20) 
>>> import pypowermate
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/user/powermate/pypowermate-master/pypowermate/__init__.py", line 1, in <module>
    from pypowermate import Powermate, PowermateTimeoutException
ImportError: cannot import name 'Powermate'
```


If there is `#nothing`, no error :
```Python 3.4.2 (default, Oct  8 2014, 10:45:20) 
>>> import pypowermate
>>>
```
It's is quite not perfect, but it works.


Think the solution is to put `from pypowermate.powermate import Powermate, PowermateTimeoutException` in __init.py :
```Python 3.4.2 (default, Oct  8 2014, 10:45:20) 
>>> import pypowermate 
>>> p=pypowermate.Powermate('/dev/input/by-id/usb-Griffin_Technology__Inc._Griffin_PowerMate-event-if00')
>>>
```

Work fine !

Reply:
This was caused by an incompatibility Python 2 vs 3, it's now fixed on the master branch.